Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/activemerchant/active_merchant
-  revision: 4c07a88589e716ae17cbe07439db695c41b66c64
+  revision: d7ce1dcd2ffbb529d038c5c094d987a99e954595
```

Comment:
I think we can use the published one in rubygems.org instead of tracking the upstream

Reply:
https://github.com/3scale/active_merchant-adyen12/blob/df39a163adaaa92e277993a394ed46ded8dda871/test/test_helper.rb#L18 depends on test helper files from https://github.com/activemerchant/active_merchant (not packed by RubyGems). I think we have to keep fetching from GH 😕.

Reply:
Right because the Gemfile is only used on dev or by the CI ... not when installing in.
👍 My fault